include/RAJA/policy/sycl/reduce.hpp

```diff
@@ -107,11 +107,11 @@ struct Reduce_Data
   Reduce_Data(T initValue, T identityValue, Offload_Info &info)
       : value(initValue)
   {
-    cl::sycl::queue* q = ::camp::resources::Sycl::get_default().get_queue();
+    ::sycl::queue* q = ::camp::resources::Sycl::get_default().get_queue();
 
 
-    device = reinterpret_cast<T *>(cl::sycl::malloc_device(sycl::MaxNumTeams * sizeof(T), *(q)));
-    host = reinterpret_cast<T *>(cl::sycl::malloc_host(sycl::MaxNumTeams * sizeof(T), *(q)));
+    device = reinterpret_cast<T *>(::sycl::malloc_device(sycl::MaxNumTeams * sizeof(T), *(q)));
+    host = reinterpret_cast<T *>(::sycl::malloc_host(sycl::MaxNumTeams * sizeof(T), *(q)));
```

Comment:
The references to `MaxNumTeams` look slightly confusing; should they be under the `RAJA::sycl::` namespace?

```suggestion
    device = reinterpret_cast<T *>(::sycl::malloc_device(RAJA::sycl::MaxNumTeams * sizeof(T), *(q)));
    host = reinterpret_cast<T *>(::sycl::malloc_host(RAJA::sycl::MaxNumTeams * sizeof(T), *(q)));
```

Reply:
Note that I did not write this code and was fixing compiler warnings. It looks like this file was started as a copy of the similarly named openmp target file.  

This is probably a better was to define the variable: https://github.com/LLNL/RAJA/blob/develop/include/RAJA/policy/openmp_target/reduce.hpp#L70

However, that file also contains similar namespace scoping issues.

I'll make an issue and we can fix those files and others for consistency as needed in a separate PR.